samples/openthread/cli/common.cmake

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# Copy new OpenThread libraries, headers, text file with the configuration
+# to the nrfxlib openthread folder.
+function(ot_libs_substitute)
+  execute_process(
+    COMMAND           git rev-parse HEAD
+    WORKING_DIRECTORY ${openthread_SOURCE_DIR}
+    RESULT_VARIABLE   git_return
+    OUTPUT_VARIABLE   openthread_git_hash
+    )
+      
+  list(APPEND OT_SETTINGS "OpenThread_commit=${openthread_git_hash}\n\n")
+      
+  # Store all OT related variables
+  get_cmake_property(_variableNames VARIABLES)
+  list (SORT _variableNames)
+  foreach (_variableName ${_variableNames})
+    if("${_variableName}" MATCHES "^CONFIG_OPENTHREAD_.*|^OT_.*") 
```

Comment:
won't this include OT_SETTINGS variable as well?

Reply:
good point ;)

It will.